docs/guides/deploy.md

```diff
@@ -46,25 +46,26 @@ Then you will need to deploy each service, with configuration so that they conne
     - For a production grade network, we suggest 3 nodes if run by the same entity, or 5 nodes if run by separate entities
     - For maximum scaleability, run using bare-metal servers.
     - Please <a href="https://tonomy.io/contact" target="_blank">contact us</a> for assistance running a production Antelope network or setting up governance.
-
-2. Create the `id.tmy` account on the blockchain
-3. Deploy the <a href="https://github.com/Tonomy-Foundation/Tonomy-Contracts/tree/master/contracts/id.tmy" target="_blank">id.tmy</a> contract to the `id.tmy` account on the blockchain
-4. [Register your applications](../../start/register-app) that you wish to connect to Tonomy ID
-5. Configure the software with the `config.json` file in the repository so that they are connected correctly using your domains, and to white-label the applications:
+2. Create the `onocoin.tmy` account on the blockchain
+3. Deploy the <a href="https://github.com/Tonomy-Foundation/Tonomy-Contracts/tree/master/contracts/onocoin.tmy" target="_blank">onocoin.tmy</a> contract to the `onocoin.tmy` account on the blockchain
+4. Create the `id.tmy` account on the blockchain
```

Comment:
after step 3, need to create and issue the new currency from the onocoin.tmy token contract